### PR TITLE
{bio}[foss/2020b] CheckM v1.0.18 w/ Python 2.7.18

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires ~~#12718~~ (matplotlib), ~~#12719~~ (DendroPy)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 3 out of 3 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/eb0a6b4594c628870eeaba365b9b4763 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 7 out of 7 (1 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.2
See https://gist.github.com/775dbd8b2977ee9ec46539df55f0e5c0 for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/c/CheckM/CheckM-1.0.18-foss-2020b-Python-2.7.18.eb [hunk 1]
```diff
@@ -0,0 +1,59 @@
+easyblock = 'PythonPackage'
```
Comment (verdurin) [inline easybuild/easyconfigs/c/CheckM/CheckM-1.0.18-foss-2020b-Python-2.7.18.eb:1]: Please re-instate the author credit.

**Review by verdurin — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3503.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/313f5717115c744ab0993da617a866bb for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3125.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/5df0b8c8f335d621ab849cc3bc88f5fa for a full test report.

**Pull request merged**